file_util/src/file.cc

```diff
@@ -0,0 +1,140 @@
+#include <leatherman/file_util/file.hpp>
+#include <boost/nowide/fstream.hpp>
+#include <boost/nowide/cenv.hpp>
+#include <boost/system/error_code.hpp>
+#include <sstream>
+#include <leatherman/logging/logging.hpp>
+
+#include <wordexp.h>
```

Comment:
This was added by @parisiale to the cthun-agent. It does not work on Windows. What would be a good alternative/fix/workaround?